core/arch/arm/include/optee_ffa.h

```diff
@@ -0,0 +1,202 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2019, Linaro Limited
+ */
+
+/*
+ * This file is exported by OP-TEE and is in kept in sync between secure
+ * world and normal world kernel driver. We're using ARM FF-A 1.0 EAC
```

Comment:
I would remove "kernel driver" to be more generic, consider normal world bootloader for example.

Reply:
Agree, I'll fix